### DISCUSSION TITLE
version 1.0.17 issue with RedirectToCanonicalUrlAttribute

### DISCUSSION THREAD

**Discussion by MCKRUZ** [Dotnet-Boxed/Templates#19] [Q&A]
I'm getting a "does not contain constructor for 3 arguments"

**Comment by RehanSaeed** [comment]
Yes. I don't make these types of breaking changes often but I did make a note in the release notes about this. The ignored controllers parameter was no longer required, so it was taken out.

One thing you will need to do is put a trailing slash '/' at the end of the URL's in your web.config file under the httpErrors section.
